.github/workflows/notify-release.yml

```diff
@@ -9,12 +9,9 @@ jobs:
   setup:
```

Comment:
please name this workflow after the file name. either notify-pending-release or notify-release, but avoid inconsistent naming.

in addition to this workflow, we need a workflow which runs eslint and tests automatically. inspire from how we do the same in other workflows